### PR TITLE
chore: decrease redis sentinel getMasterTimeout

### PR DESCRIPTION
**Pull request by rokroskar**
Lowers the `redis.sentinel.getMasterTimeout` to avoid having redis start very slowly. 

/deploy

### CODE REVIEW

File: helm-chart/renku/values.yaml [hunk 1]
```diff
@@ -374,6 +374,7 @@ redis:
       appendonly no
       # Disable RDB persistence.
       save ""
+
```
Comment (aledegano) [inline helm-chart/renku/values.yaml:377]: not that is super important, but was this new line intended?

Comment (rokroskar) [reply helm-chart/renku/values.yaml:377]: no...

Comment (rokroskar) [reply helm-chart/renku/values.yaml:377]: haha now I'm confused... I removed it but somehow it's still there 🤷

Comment (aledegano) [reply helm-chart/renku/values.yaml:377]: there's also 2 white spaces?...

Comment (rokroskar) [reply helm-chart/renku/values.yaml:377]: 🤦

**Review by seanrmurphy — APPROVED** [review]

**Review by Panaetius — APPROVED** [review]

**Pull request merged**